CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@
 4) Run `git checkout .idea` in the Terminal to regain the codeStyles
 5) If you ever want to compile the project with IDEA itself, you have to open
   `File > Settings > Java Compiler > Additional command line parameters` and insert  
-  `-Xlint:unchecked --add-exports javafx.controls/com.sun.javafx.scene.control=ALL-UNNAMED --add-exports javafx.controls/com.sun.javafx.scene.control.skin=ALL-UNNAMED --add-exports javafx.web/com.sun.webkit=ALL-UNNAMED`
```

Comment:
It would be better to just mention that javac, kotlinc and JVM all need certain arguments, defined in gradle.project file. The developer should be able to find that information on his own. It is better than the possibility that the information we provide is out of date.